source/code/include/playfab/PlayFabEventApi.h

```diff
@@ -15,6 +15,9 @@ namespace PlayFab
     {
     public:
         PlayFabEventAPI(); // Default constructor
```

Comment:
It is unclear to me what the default behavior is.  The new ctor takes a bool that would seemingly dictate the underlying behavior of the class.  The default ctor appears does what, then?  Should we just have the new ctor with a default parameter?

Reply:
I was initially thinking that too, Scott's implementation passes true directly in the cpp constructor implementation, so I opted for 1:1 as much as I could (there were a few places where we made changes that may not have made it back to bumblelion).

Reply:
I don't think I'd mind a ctor with a default parameter if that makes this more clear for y'all
`PlayFabEventAPI(bool threadedEventPipeline=true);`

The reason I made the change without one was to try and keep things additive and our team just general tries to avoid default parameters. Also since y'all expose the source it's not difficult to figure out what the default ctor does (though I agree with the sentiment that looking at source files to figure out this behavior is subpar)

Reply:
though I do appreciate you keeping the change as 1:1 as possible :)